src/main/java/arrays/homework_3/ani_darbinyan/SortingAlgorithms.java

```diff
@@ -0,0 +1,10 @@
+package arrays.homework_3;
+
+public class SortingAlgorithms {
+   /* TODO: Write algorithms below
+       1. Insertion Sort
+       2. Selection Sort
+       3. Bubble Sort
+    */
+
```

Comment:
I dont see the implementation

Reply:
I have added the implementation, please, check the changes I have made